### PR TITLE
Bundle both mixer and esp in one container

### PR DESCRIPTION
**Pull request by shifucun**
This tests the feasibility to bring multiple process in the same container to prepare for doing this for mixer + website.

This PR also allows reading the api secret from environment directly, so a custom mixer can be brought up locally without GCP at all.

### CODE REVIEW

File: internal/util/util.go [hunk 1]
```diff
@@ -503,23 +505,44 @@ func StringListIntersection(list [][]string) []string {
...
 }
 
-// MapsClient gets the client for Maps API.
-func MapsClient(ctx context.Context, projectID string) (*maps.Client, error) {
-	secretClient, err := secretmanager.NewClient(ctx)
+func ReadLatestSecret(
+	ctx context.Context, projectID, secretID string,
+) (string, error) {
+
+	secret := os.Getenv(strings.Replace(secretID, "-", "_", -1))
```
Comment (pradh) [inline internal/util/util.go:512]: nit: add a note why Replace?

File: scripts/push_binary.sh [hunk 2]
```diff
@@ -30,5 +30,5 @@ fi
 gcloud builds submit \
     --project=datcom-ci \
     --config=build/ci/cloudbuild.push.yaml \
-    --substitutions SHORT_SHA="$TAG" \
+    --substitutions SHORT_SHA="$TAG",_SKIP_AUTOPUSH_UPDATE=true \
```
Comment (pradh) [inline scripts/push_binary.sh:33]: nit: should an external var not start with `_`?

**Review by pradh — APPROVED** [review]
Nice!

File: scripts/push_binary.sh (hunk 2 above)
Comment (shifucun) [reply scripts/push_binary.sh:33]: cloudbuild requires custom variable to start with _: https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#using_user-defined_substitutions

File: internal/util/util.go (hunk 1 above)
Comment (shifucun) [reply internal/util/util.go:514]: Done

**Review by shifucun — COMMENTED** [review]
thanks!

**Pull request merged**